stdlib/email/src/main/java/org/ballerinalang/stdlib/email/client/EmailAccessClient.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.stdlib.email.client;
+
+import org.ballerinalang.jvm.BallerinaErrors;
+import org.ballerinalang.jvm.StringUtils;
+import org.ballerinalang.jvm.values.MapValue;
+import org.ballerinalang.jvm.values.ObjectValue;
+import org.ballerinalang.stdlib.email.util.EmailAccessUtil;
+import org.ballerinalang.stdlib.email.util.EmailConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import javax.mail.Folder;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.NoSuchProviderException;
+import javax.mail.Session;
+import javax.mail.Store;
+
+/**
+ * Contains the functionality of email reading with POP and IMAP clients.
+ *
+ * @since 1.2.0
+ */
+public class EmailAccessClient {
+
+    private static final Logger log = LoggerFactory.getLogger(EmailAccessClient.class);
+
+    private EmailAccessClient() {
+        // A Singleton class.
+    }
+
+    /**
+     * Initializes the ObjectValue object with the POP properties.
+     * @param clientEndpoint Represents the POP Client class
+     * @param host Represents the host address of POP server
+     * @param username Represents the username of the POP server
+     * @param password Represents the password of the POP server
+     * @param config Properties required to configure the POP session
+     * @return If an error occurs in the POP client, returns an error
+     */
+    public static Object initPopClientEndpoint(ObjectValue clientEndpoint, String host, String username,
+                                               String password, MapValue<Object, Object> config) {
+        Properties properties = EmailAccessUtil.getPopProperties(config, host);
+        Session session = Session.getInstance(properties, null);
+        try {
+            Store store;
```

Comment:
Shall we consolidate this to a single line?

Reply:
Consolidated.